runtime/onert/frontend/circle/src/circle_loader.cc

```diff
@@ -91,7 +91,7 @@ class CircleLoader final : public base_loader::BaseLoader<LoaderDomain, CircleLo
     }
   }
 
-  std::unique_ptr<ir::Graph> loadSubgraph(const circle::SubGraph *circle_subg)
+  std::unique_ptr<ir::Graph> loadSubgraph(const circle::SubGraph *circle_subg) override
```

Comment:
Should be private

Reply:
Thanks for your pointing out. It's my mistake.